doculab/docs/mailchimp.textile

```diff
@@ -0,0 +1,73 @@
+Our MailChimp integration will automatically sync Chargify customers to a MailChimp list.
+
+**This integration is currently in beta status**.  "What does it mean for a Chargify feature to be beta?":/beta-label
+
+h2. Availability
+
+The MailChimp integration is available to merchants:
+
+* On the free Developer plan (for testing), OR
+* On the Small Business Plus plan or higher
+
+If you would like to test out the integration during the beta period, but you do not meet these criteria, you may "open a support ticket":https://chargify.zendesk.com/hc/en-us/requests/new and request a one month credit for a free upgrade to the Small Business Plus plan.
+
+h2. Configuring the Integration
+
+The MailChimp integration tab is found within the main Integrations tab on each of your Sites:
+
+!/images/doculab/mailchimp-api-key-1.png!
+
+h3. API Key
+
+After you enter your <a href="http://kb.mailchimp.com/accounts/management/about-api-keys">MailChimp API Key</a>, click the 'check' link to verify that it is correct.
+
+!/images/doculab/mailchimp-api-key-check-2.png!
+
+You should see the "Everything's Chimpy!" message appear below your key.
+
+Once that's done, click the "Continue to List Configuration" button.
+
+h4. Choose List
+
+Next you will need to choose which MailChimp list to sync to.
+
+!/images/doculab/mailchimp-choose-list-3.png!
+
+Use the drop-down to select one of your MailChimp lists.  Remember that this needs to be a new, empty list.
+
+When you're ready, click the "Continue to Final Setup" button.
+
+h4. Start Exporting
+
+!/images/doculab/mailchimp-start-exporting-4.png!
+
+Click the "Start Exporting" button to begin the process.
+
+h2. How it Works
+
+Once you click the "Start Exporting" button, Chargify will sync the 
+
+* email address,
+* first name,
+* last name,
+* customer ID,
+* current product, and
+* subscription state
+
+of all of the customers in your Chargify Site to the MailChimp list you selected.
+
+Going forward, we will sync changes when a customer is updated, a new subscription is added, or a subscription is changed.
+
+h2. Making Changes
+
+Once the MailChimp integration is enabled, you have the option to disable it in order to make changes, or to completely start over.
```

Comment:
I think we want to inform the merchant that completely start over does not: 1) delete the MailChimp list. 2) does not remove emails from the MailChimp list. The merchant should also select a new list or clear the existing list when doing this.